firefox-ios/Client/Frontend/Share/URLActivityItemProvider.swift

```diff
@@ -4,32 +4,64 @@
 
 import Foundation
 import UniformTypeIdentifiers
+import Shared
 
 class URLActivityItemProvider: UIActivityItemProvider, @unchecked Sendable {
+    private struct ActivityIdentifiers {
+        static let whatsApp = "net.whatsapp.WhatsApp.ShareExtension"
+    }
+
+    private static var isSentFromFirefoxTreatmentA: Bool {
+        return LegacyFeatureFlagsManager.shared.isFeatureEnabled(.sentFromFirefoxTreatmentA, checking: .buildOnly)
+    }
+
     private let url: URL
+    private let allowSentFromFirefoxTreatment: Bool // FXIOS-9879 For the Sent from Firefox experiment
```

Comment:
super nit: i prefer to have comments on top, so i don't forget to read it. But super nit

Reply:
I usually agree too. In this specific case, I think these are ok here since I just want the FXIOS-9879 in there for finding lines to "clean up" later when the Sent from Firefox experiment is complete (it's not intended for documentation in this case 😅 ).

Reply:
Ok that makes sense, right. 👍